src/test/resources/application.yml

```diff
@@ -358,4 +358,8 @@ welsh:
     9: "Medi"
     10: "Hydref"
     11: "Tachwedd"
-    12: "Rhagfyr"
\ No newline at end of file
+    12: "Rhagfyr"
+
+ld:
+  client_key: changeMe
+  user_key: changeMe
```

Comment:
Does this work when developing locally? I've not used launch darkly before so am curious. ta